variables.tf

```diff
@@ -77,6 +77,7 @@ variable "instances" {
       name  = string
       value = string
     })), [])
+    environmentd_extra_args = optional(list(string), [])
```

Comment:
Oh neat! Is this what we'd uses to set, for example, `enable_create_table_from_source = true;` or something else?

Reply:
exactly!

```
environmentd_extra_args:
  - --system-parameter-default=enable_create_table_from_source=true
```

There's a side effort I've got to let us do this via config files, that would allow us to do this without restarting environmentd, but it's not ready yet.